nodejs/week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+// const app = express();
+const router = express.Router()
+
+const mealsArray = require('../data/meals.json');
+
+     
+// ++++++++++++++++++++++++ Testing for combining the whole meal search commands into one ++++++++++++++++++++++++  
+
+router.get('/meals', (req, res)=>{
+        
+// ===================================================================================  for id :  /meals?id=2
+      const givenid = parseInt(req.query.id);
+      
+        let mealscount = Object.keys(mealsArray).length;

```

Comment:
1. `mealscount` never changes, better make it a const.
2. It has to be named using camel style.
3. As you use it only after line 56, it's better to move it there.